02_configure_host.sh

```diff
@@ -21,11 +21,13 @@ early_deploy_validation
 manage_libvirtd() {
   case ${DISTRO} in
       centos9|rhel9)
-          for i in qemu network nodedev nwfilter secret storage interface; do
+          for i in qemu interface network nodedev nwfilter secret storage; do
               sudo systemctl enable --now virt${i}d.socket
               sudo systemctl enable --now virt${i}d-ro.socket
               sudo systemctl enable --now virt${i}d-admin.socket
           done
+          sudo systemctl enable virtproxyd.service
```

Comment:
I'm not familiar with this component, but normally you enable either the service or the activation socket, not both.

Reply:
True, it is either the socket or service. Also, we are enabling the daemon but not starting it.  I can remove this step.